src/modules/battery.cpp

```diff
@@ -74,6 +74,8 @@ std::tuple<uint16_t, std::string> waybar::modules::Battery::getInfos()
       std::ifstream(bat / "status") >> _status;
       if (_status != "Unknown") {
         status = _status;
+      }else if (config_["full-is-unknown"].isString() && config_["full-is-unknown"] == "true") {
```

Comment:
I don't really like this kind of option, I think it's better to do that for example "full-capacity": 98, what do you think?
PS: also a space is missing next to the bracket

Reply:
Sorry for the formatting errors.
Problem is, full-capacity does not allow me to change the format, no? So I would have the charging format, since that is the default if status is Unknown.

Reply:
Okay, how's this:
state and status are now completely independent. A class is set for both (if configured).
There's the possibility for format-[status]-[state], format-[status] and format-[state] (and the default format, of course).

Reply:
Which allows people to do with unknown what they want.

Reply:
Here's an example config:
```
    "battery": {
        "bat": "BAT0",
        "states": {
            "full": 100,
            "good": 90,
            "warning": 30,
            "critical": 15
        },
        "format-charging-full": "🔌[full]",
        "format-unknown-full": "🔌[full]",
        "format-charging": "🔌{capacity}% {icon}",
        "format-discharging": "🔋{capacity}% v {icon}",
        "format": "🔋{capacity}% {icon}",
        "format-icons": [" ","▁","▂","▃","▄","▅","▆","▇","█"]
    },
```

Reply:
Love it!